src/condor_tools/htcondor_cli/eventlog.py

```diff
@@ -68,40 +91,150 @@ def __init__(self, logger, log_file, **options):
+                    elif event_summary.get("Return Value") != 0:
+                        site_stats[site]["Failures"] = site_stats[site].get("Failures") + 1
+                # convert wall time, good time, and cpu usage to time format
+                for site, stats in site_stats.items():
+                    wall_time = f"0+{stats['Wall Time']//3600:02d}:{(stats['Wall Time']//60)%60:02d}:{stats['Wall Time']%60:02d}"
+                    site_stats[site]["Wall Time"] = wall_time
+                    good_time = f"0+{stats['Good Time']//3600:02d}:{(stats['Good Time']//60)%60:02d}:{stats['Good Time']%60:02d}"
+                    site_stats[site]["Good Time"] = good_time
+                    cpu_usage = f"0+{stats['CPU Usage']//3600:02d}:{(stats['CPU Usage']//60)%60:02d}:{stats['CPU Usage']%60:02d}"
+                    site_stats[site]["CPU Usage"] = cpu_usage
+                # if json output, print json
+                if options.get("json", True):
+                    print(json.dumps(site_stats, indent=4))
+                elif options.get("csv", True):
+                    # print header
+                    print("Site,Evictions,Wall Time,Good Time,CPU Usage,Job Starts,Job Successes,Job Failures")
+                    # print stats for each site
+                    for site, stats in site_stats.items():
+                        if site:
+                            print(f"{site},{stats['Evictions']},{stats['Wall Time']},{stats['Good Time']},{stats['CPU Usage']},{stats['Starts']},{stats.get('Successes', '')},{stats.get('Failures', '')}")
+                else:
+                # print header
+                    print(f"{'Site':<20} {'Evictions':<12} {'Wall Time':<12} {'Good Time':<12} {'CPU Usage':<12} {'Job Starts':<12} {'Job Successes':<14} {'Job Failures':<12}")
+                    # print stats for each site
+                    for site, stats in site_stats.items():
+                        if site:
+                            print(f"{site:<20} {stats['Evictions']:<12} {stats['Wall Time']:<12} {stats['Good Time']:<12} {stats['CPU Usage']:<12} {stats['Starts']:<12} {stats.get('Successes', ''):<14} {stats.get('Failures', ''):<12}")
+        elif options.get("json", True) or options.get("csv", True):
+            # add event summaries as objects to list for JSON output
+            json_output = []
+            for job_id, event_summary in event_summaries.items():
+                # convert cpu usage, wall time, and good time to seconds
+                cpu_usage = event_summary.get("CPU Usage", "0+00:00:00")
+                cpu_usage_seconds = int(cpu_usage.split("+")[1].replace(":", ""))
+                wall_time = event_summary.get("Wall Time", "0+00:00:00")
+                wall_time_seconds = int(wall_time.split("+")[1].replace(":", ""))
+                good_time = event_summary.get("Good Time", "0+00:00:00")
+                good_time_seconds = int(good_time.split("+")[1].replace(":", ""))
```

Comment:
If a job which ran for one hour, one minutes, and one second has a usage of `01:01:01`, this code says it ran for 10,101 seconds, which is 2 hours, 48 minutes, and 21 seconds.
If that runtime had been instead represented as `1:1:1`, this code says it ran for 111 seconds, which is 0 hours, 1 minute, and 51 seconds.